service/src/main/java/greencity/security/service/OwnSecurityServiceImpl.java

```diff
@@ -208,6 +221,15 @@ public SuccessSignInDto signIn(final OwnSignInDto dto) {
         return new SuccessSignInDto(user.getId(), accessToken, refreshToken, user.getName(), true);
     }
 
+    private void publishFailureEvent(String email, String password, String errorMessage) {
+        UsernamePasswordAuthenticationToken auth =
+            new UsernamePasswordAuthenticationToken(email, password);
```

Comment:
Why do you need to publish an event with a user password?
It is sensitive information and should not be logged